spec-main/chromium-spec.ts

```diff
@@ -1233,15 +1233,21 @@ describe('chromium features', () => {
       w.loadURL(pdfSource);
       const [, contents] = await emittedOnce(app, 'web-contents-created');
       expect(contents.getURL()).to.equal('chrome-extension://mhjfbmdgcfjbbpaeojofohoefgiehjai/index.html');
-      await emittedOnce(contents, 'did-finish-load');
+      await new Promise((resolve) => {
+        contents.on('did-finish-load', resolve);
+        contents.on('did-frame-finish-load', resolve);
```

Comment:
`did-finish-load` is just a special case of `did-frame-finish-load`, they are emitted in the same C++ method, I think only listening to `did-frame-finish-load` is enough.

Reply:
@zcbenz that is what I had originally, but for some reason on Windows, it was failing if I didn't listen for `did-finish-load`, but on ARM failing if I didn't listen for `did-frame-finish-load`.